content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1154 @@
+    },
+    style: function(feature) {
+      switch (feature.properties.accident_severity) {
+          case "Serious": return {color: "#FEB24C"};
+          case "Fatal":   return {color: "#BD0026"};
+       }
+    }
+  }).addTo(map);
+  map.fitBounds(layer.getBounds());
+  var legend = L.control({position: "bottomright"});
+  legend.onAdd = function (map) {
+    var div = L.DomUtil.create("div", "info legend"), labels = [];
+    labels.push("<h6>accident_severity</h6>");
+    labels.push("<i style='background:#FEB24C'></i>Serious");
+    labels.push("<i style='background:#BD0026'></i>Fatal");
+    div.innerHTML = labels.join("<br>");
+    return div;
+  };
+  legend.addTo(map);
+  var info = L.control();
+  info.onAdd = function (map) {
+    this._div = L.DomUtil.create("div", "info");
+    this.update();
+    return this._div;
+  };
+  info.update = function (props) {
+    this._div.innerHTML = "<h6>Crashes in West Yorkshire (2017)</h6>";
+  };
+  info.addTo(map);
+</script>
+
+<style>
+.info { padding: 6px 8px; font: 14px/16px Arial, Helvetica, sans-serif; background: white; background: rgba(255,255,255,0.8); box-shadow: 0 0 15px rgba(0,0,0,0.2); border-radius: 5px; } .info h4 { margin: 0 0 5px; color: #777; }
+.legend { text-align: left; line-height: 18px; color: #555; } .legend i { width: 18px; height: 18px; float: left; margin-right: 8px; opacity: 0.7; }</style>
+
+</div>
+
+<!--/html_preserve-->
+
+## Conclusion
+
+[`stats19`](https://github.com/ropensci/stats19) provides access to a
+reliable and official road safety dataset. As covered in this post, it
+helps with data discovery, download, cleaning and formatting, allowing
+you to focus on the real work of analysis (see the package’s
+introductory
+[vignette](https://itsleeds.github.io/stats19/articles/stats19.html) for
+more on this). In our experience, 80% of time spent using STATS19 data
+was spent on data cleaning. Hopefully, now the data is freely available
+in a more useful form, 100% of the time can be spent on analysis\! We
+think it could help many people, especially, including campaigners,
+academics and local authority planners aiming to make the roads of the
+UK and the rest of the world a safe place for all of us.
+
+There are many possible next steps, including:
+
+  - Comparing these datasets with interventions such as 20 mph zones
+    (Grundy et al. 2009) and links with street morphology (Sarkar,
+    Webster, and Kumari 2018).
+  - The creation of more general software for accessing and working with
+    road crash data worldwide.
+  - Making the data even more available by provide the data as part of
+    an interactive web application, a technique successfully used in the
+    Propensity to Cycle Tool (PCT) project hosted at
+    [www.pct.bike/](http://www.pct.bike/) (this would likely take
+    further times/resources beyond what we can provide in our spare
+    time\!).
+
+For now, however, we want to take the opportunity to celebrate t
```

Comment:
@stefaniebutland my apologies for missing this. I am not quite sure how Hugo/custom rOpenSci HTML does this. I will try to resolve it but just in case you see this before I waste more time. As you know the Rmd generates the refs and we just name the refs. I know this means using "HTML bookmarks". 

But trying few combinations did not work.

Reply:
I can see that in this [article](http://localhost:1313/blog/2019/01/29/rdhs/#fn:1) for example, output CSS uses`footnote-ref` class is used. The "footnotes" are a `ul` which I would assume need to be generated from a style such as the one you have suggested (`[^1] Lovelace ...`).

But I just don't know why I cannot get it to work. Tried:

`[^1] Lovelace ...`
`[^1]Lovelace ...`
`[^1] - Lovelace ...`
`- [^1] Lovelace ...`
` - [^1] Lovelace ...`

etc.

Reply:
got it, will push soon.

Reply:
the style should be `[^1]: Reference One...` etc.

Reply:
@layik Please accept my apology! I omitted the colon in [^1]:. I should have linked to an example instead of typing it